astrokat/observe_main.py

```diff
@@ -122,13 +122,48 @@ def read_targets(target_items):
     return target_list
 
 
+def slew(session, target_info):
+    """Simple way to get telescope to slew to target
+
+    Parameters
+    ----------
+    session: `CaptureSession` object
+    target_info: dictionary with target observation info
+
+    """
+    katpt_tgt = target_info["target"]
+
+    user_logger.info("Slewing to first target")
+    session.set_target(katpt_tgt)
+    session.activity('slew')
+    # Start moving each antenna to the target
+    if session.kat.dry_run:
+        # Apply average slew time
+        session._slew_to(katpt_tgt)
+    else:
+        # Start moving each antenna to the target
+        session.ants.req.mode('POINT')
+
+    # Initiate FBFUSE proxy settings from mkat-sessions capture_start
+    if session.fbf:
+        user_logger.info('Initiating FBF beam complete')
+        if not session.fbf.check_provisioning_beams_complete():
+            raise session.FBFUSEError("Provision beams not ready")
+
+    # Wait until a quorum is in position (with timeout)
+    if not session.kat.dry_run:
+        session.wait(session.ants, 'lock', True, timeout=300, quorum=session.quorum)
+
+    user_logger.info('target reached')
+
```

Comment:
`session.wait` returns a boolean:  True if a quorum of antennas reached the target in time.  The user log here should be dependent on that return value.

And do we want the observation to continue if we couldn't get to the first target?

Reply:
Indeed an oversight, I corrected this and made the log message dependent on a return `True` value.
Since `session.wait` will issue a warning, I am not going to take any action toward interrupting the observation.
The `on_target` test in the `tract` command that will follow should either allow the quorum to be reached, or it will react appropriately.